src/services/swap_service.ts

```diff
@@ -364,7 +364,9 @@ export class SwapService {
         // Cannot eth_gasEstimate for /price when RFQ Native liquidity is included
         const isNativeIncluded = swapQuote.sourceBreakdown.Native !== undefined;
         const isQuote = endpoint === 'quote';
-        const canEstimateGas = isQuote || !isNativeIncluded;
+        const canEstimateGas =
+            (isQuote && swapQuote.orders.every((o) => o.type !== FillQuoteTransformerOrderType.Otc)) ||
```

Comment:
hmmmm..... I forget why I added this check. Note to self to double check if this can be removed